include/deal.II/sundials/sundials_backport.h

```diff
@@ -29,8 +29,9 @@
 #include <deal.II/base/config.h>
 
 #ifdef DEAL_II_WITH_SUNDIALS
+#  if DEAL_II_SUNDIALS_VERSION_LT(5, 0, 0)
```

Comment:
The only time `sundials_backport.h` is included in any deal.II header is guarded with `DEAL_II_SUNDIALS_VERSION_LT(5, 0, 0)`.